x-pack/plugins/saved_objects_tagging/common/constants.ts

```diff
@@ -20,4 +20,17 @@ export const tagManagementSectionId = 'tags';
 /**
  * The list of saved object types that are currently supporting tagging.
  */
-export const taggableTypes = ['dashboard', 'visualization', 'map', 'lens', 'search'];
+export const taggableTypes = [
+  'dashboard',
+  'visualization',
+  'map',
+  'lens',
+  'search',
+  // Duplicated types are needed for Fleet integration assets
+  'osquery_pack',
+  'osquery_pack_asset',
+  'osquery_saved_query',
+  'osquery-pack',
+  'osquery-pack-asset',
+  'osquery-saved-query',
```

Comment:
Why are the additions to this list different than the ones in the `schema.ts` file?